code-names-common/src/main/java/com/codenames/enums/Language.java

```diff
@@ -0,0 +1,13 @@
+package com.codenames.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Getter
+public enum Language {
+    UA("ua"),
+    EN("en");
+
+    private final String strType;
```

Comment:
Better go with some more descriptive naming like `languageCode`

Reply:
Fixed this